README.md

```diff
@@ -174,6 +174,15 @@ jobs:
         run: pub run test
 ```
 
+## Cached example
+
+```yml
+      - uses: dart-lang/setup-dart@v1.3
+        with:
+          sdk: '2.9.0'
+          cache: true # Should only be used when a fixed SDK version is specified
```

Comment:
Why only with a fixed version of the SDK? Otherwise, we may cache and use version x, even when x+1 becomes available?

Reply:
Yes